docs/source/zero-code-change.rst

```diff
@@ -0,0 +1,209 @@
+anything which is implemented in Scikit-Learn but not yet accelerated by cuML.
+
+Additionally, ``cuml.accel`` offers a quick way to evaluate the minimum
+acceleration cuML can provide for your workload without touching a line of
+code.
+
+2. Why use cuML directly instead of cuml.accel?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In many cases, ``cuml.accel`` offers enough of a performance boost on its own
+that there is no need to migrate code to cuML. However, cuML's API offers a
+variety of additional parameters that let you fine-tune GPU execution in order
+to get the maximum possible performance out of NVIDIA GPUs. So for software
+that will always be run with NVIDIA GPUs available, it may be worthwhile to
+write your code directly with cuML.
+
+Additionally, running code directly with cuML offers finer control over GPU
+memory usage. ``cuml.accel`` will automatically use `unified or managed memory <https://developer.nvidia.com/blog/unified-memory-cuda-beginners/>`_
+for allocations in order to reduce the risk of CUDA OOM errors. In
+contrast, cuML defaults to ordinary device memory, which can offer improved
+performance but requires slightly more care to avoid exhausting the GPU VRAM.
+If you experience unexpectedly slow performance with ``cuml.accel``, you can
+try disabling the use of unified memory with the ``--disable-uvm`` flag.
+
+3. What does ``cuml.accel`` accelerate?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``cuml.accel`` is designed to provide zero code change acceleration of any
+Scikit-Learn-like estimator which has an equivalent cuML implementation,
+including estimators from Scikit-Learn, UMAP-Learn, and HDBSCAN. Currently,
+the following estimators are mostly or entirely accelerated when run under
+``cuml.accel``:
+
+* UMAP-Learn
+    * ``umap.UMAP``
+* HDBSCAN
+    * ``hdbscan.HDBSCAN``
+* Scikit-Learn
+    * ``sklearn.cluster.KMeans``
+    * ``sklearn.cluster.DBSCAN``
+    * ``sklearn.decomposition.PCA``
+    * ``sklearn.decomposition.TruncatedSVD``
+    * ``sklearn.ensemble.RandomForestClassifier``
+    * ``sklearn.ensemble.RandomForestRegressor``
+    * ``sklearn.kernel_ridge.KernelRidge``
+    * ``sklearn.linear_model.LinearRegression``
+    * ``sklearn.linear_model.LogisticRegression``
+    * ``sklearn.linear_model.ElasticNet``
+    * ``sklearn.linear_model.Ridge``
+    * ``sklearn.linear_model.Lasso``
+    * ``sklearn.manifold.TSNE``
+    * ``sklearn.neighbors.NearestNeighbors``
+    * ``sklearn.neighbors.KNeighborsClassifier``
+    * ``sklearn.neighbors.KNeighborsRegressor``
+
+This list will continue to expand as ``cuml.accel`` development
+continues to cover all algorithms present in cuML.
+Please see `Zero Code Change Limitations <0cc_limitations.rst>`_
+for known limitations.
+
+Currently the following algorithms are next in our roadmap for the next version
+of cuml.accel:
+
+* AgglomerativeClustering
+* SVC and SVR support vector machine models.
+* IncrementalPCA
+* Naive Bayes models
```

Comment:
I'm not sure that sharing the roadmap as part of the docs is necessarily a good idea. It does not provide that much value and makes a promise we might not be able / want to keep? I would suggest to leave that out and or perhaps link to an issue instead.